string[] objects
string world_name
---
path_finding/Point[] path
